src/components/Tooltip/Tooltip.stories.module.css

```
@@ -0,0 +1,20 @@
+.trigger--spacing {
+  margin: 9rem;
```

Weird number but it's appropriate required for the tooltip to place where alignment is indicated, otherwise Tippy places somewhere else due to lack of space

---

Maybe leave that in a comment?

---

Sounds good, added comment

---

let's also do the footnote commenting thing brad had, i.e.:
```suggestion
  margin: 9rem; /* 1 */
```
to note that this specific line is connected to the 1) comment above

---

Will do